### PR TITLE
Trivial refactors

### PR DESCRIPTION
**Pull request by CismonX**
Some trivial refactors which make the code cleaner, without breaking backward compatiblilty.

Changes including: 
* Fix typos and code smell.
* Old-fashioned code rewritten with newer JDK 1.8 APIs.
* Eliminate unused code (only those `private` ones).

Code within the `org.json` package and test cases are left untouched, for now.

### CODE REVIEW

File: latke-core/src/main/java/org/b3log/latke/http/renderer/AbstractFreeMarkerRenderer.java [hunk 1]
```diff
@@ -127,11 +127,11 @@ protected String genHTML(final Request request, final Map<String, Object> dataMo
...
 
         final StringBuilder pageContentBuilder = new StringBuilder(stringWriter.toString());
-        final long endimeMillis = System.currentTimeMillis();
-        final String dateString = DateFormatUtils.format(endimeMillis, "yyyy/MM/dd HH:mm:ss");
+        final long endTimeMillis = System.currentTimeMillis();
+        final String dateString = DateFormatUtils.format(endTimeMillis, "yyyy/MM/dd HH:mm:ss");
         final long startTimeMillis = (Long) request.getAttribute(Keys.HttpRequest.START_TIME_MILLIS);
-        final String msg = String.format("\n<!-- Generated by Latke (https://github.com/88250/latke) in %1$dms, %2$s -->",
-                endimeMillis - startTimeMillis, dateString);
+        final String msg = String.format("%n<!-- Generated by Latke (https://github.com/88250/latke) in %1$dms, %2$s -->",
```
Comment (88250) [inline latke-core/src/main/java/org/b3log/latke/http/renderer/AbstractFreeMarkerRenderer.java:133]: `%n` 这里有问题哦

Comment (CismonX) [reply latke-core/src/main/java/org/b3log/latke/http/renderer/AbstractFreeMarkerRenderer.java]: 👌, reverted.

**Comment by 88250** [comment]
@CismonX 结合项目实操后有点问题，麻烦拉取 Solo 项目联编：

```
java.lang.ClassCastException: class [Ljava.lang.Object; cannot be cast to class [Ljava.lang.String; ([Ljava.lang.Object; and [Ljava.lang.String; are in module java.base of loader 'bootstrap')
        at org.b3log.solo.util.Solos.clone(Solos.java:516) ~[classes/:?]
```

**Comment by CismonX** [comment]
@88250 👌

**Comment by CismonX** [comment]
@88250 Fixed in #31

**Pull request merged**